crates/nu-command/src/formats/from/json.rs

```diff
@@ -83,7 +83,7 @@ impl Command for FromJson {
                             strict,
                             engine_state.signals().clone(),
                         ),
-                        update_metadata(metadata),
+                        metadata,
```

Comment:
I was confused at first what was going on here, but then I saw this is for `from`.